docs/source/index.rst

```diff
@@ -1,5 +1,13 @@
-Welcome to Intake's documentation!
-==================================
+Intake: A general interface for loading data
+============================================
+
+Intake is a lightweight set of tools for loading and sharing data in data science projects.
+Intake helps you:
+
+* Load data from a variety of formats (see :ref:`plugin-directory`) into containers you already use, like Pandas dataframes, Python lists, NumPy arrays, and more.
+* Convert your boilerplate data loading code into reusable Intake plugins
```

Comment:
* Load data catalogs and actual data from remote/cloud storage

Reply:
> Convert your boilerplate data loading code into reusable Intake plugins

You mean, a convenient place to write that boilerplate once and use many times?

Reply:
Yeah, going to try the word "Translate" there instead of "Convert"